src/main/features/response/views/timeline.njk

```diff
@@ -1,88 +1,27 @@
-    <span class="error-message" id="{{ dateKey }}[label]">
-      {{ t(dateError) }}
-    </span>
-    {% endif %}
-    {% if descriptionError %}
-    <span class="error-message" id="{{ descriptionKey }}[label]">
-      {{ t(descriptionError) }}
-    </span>
-    {% endif %}
-
-    <label class="form-label-bold mobile-show" for="{{ dateKey }}">{{ index + 1 }}{{ t('. Date') }}</label>
-    <span class="form-hint mobile-show">{{ t('For example, 1 May 2017') }}</span>
-    <input id="{{ dateKey }}" name="{{ dateKey }}" value="{{ dateValue if dateValue }}"
-           class="form-control multiline-column-1 {% if dateError %}form-control-error{% endif %}" type="text">
-
-    <label for="{{ descriptionKey }}" class="form-label-bold mobile-show">
-      {{ t('What happened') }}
-    </label>
-    <span class="form-hint mobile-show">{{ t('For example, you might have signed a contract') }}</span>
-    <textarea id="{{ descriptionKey }}" name="{{ descriptionKey }}"
-              class="form-control multiline-column-text {% if descriptionError %}form-control-error{% endif %}"
-              rows="2">{{ descriptionValue if descriptionValue }}</textarea>
-  </div>
-{% endmacro %}
-
 {% block content %}
 <div class="grid-row">
   <div class="column-two-thirds">
 
     <form method="post" novalidate>
       {{ csrfProtection(csrf) }}
-      <h3 class="heading-small">{{ t('Add your timeline of events (optional)') }}</h3>
+      <h2 class="heading-small">{{ t('Add your timeline of events (optional)') }}</h2>
       <p>{{ t('Include dates if you know them.') }}</p>
-      <div class="multiline">
-        <div class="multiline-headings">
-          <div class="multiline-column-1">
-            <label class="form-label-bold">
-              {{ t('Date') }}
-            </label>
-            <span class="form-hint">{{ t('For example, 1 May 2017') }}</span>
-          </div>
-          <div class="multiline-column-2">
-            <label class="form-label-bold">
-              {{ t('What happened') }}
-            </label>
-            <span class="form-hint">{{ t('For example, you might have signed a contract') }}</span>
-          </div>
-        </div>
-
-        <div class="multiline-rows">
-        {% for row in form.model.rows %}
-          {{ timelineRow(loop.index0, form) }}
-        {% endfor %}
-        </div>
-
-      </div>
 
-      {% if form.model and form.model.canAddMoreRows() %}
-        <div id="add-event-container" class="form-group">
-          {{ linkButton('Add another event', 'action[addRow]', 'button button-secondary') }}
-        </div>
-      {% endif %}
+      {{ timelineForm(form) }}
 
       <div class="panel panel-border-wide">
         <p>{{
               t('Your response will be sent to the claimant %s. A judge will not consider your response unless your case goes to a hearing.',
-                { postProcess: 'sprintf', sprintf: [ claimantName ] }) }}</p>
+                { postProcess: 'sprintf', sprintf: [ claimantName ] })
```

Comment:
can you switch this to using interpolation please? 
`{{ t('claimant {{ claimantName }}', { claimantName })`

Reply:
I will